### PR TITLE
Responsive UI for participants table

### PR DESCRIPTION
**Pull request by vidya-ram**
Responsive UI for participants table (participants check-in page)

### CODE REVIEW

File: funnel/__init__.py [hunk 1]
```diff
@@ -26,6 +26,9 @@
 assets['schedules.css'][version] = 'css/schedules.css'
 assets['schedules.js'][version] = 'js/schedules.js'
 assets['screens.css'][version] = 'css/screens.css'
+assets['footable.js'][version] = 'js/libs/footable-min.js'
+assets['footable-core.css'][version] = 'css/footable.core.css'
+assets['footable-metro.css'][version] = 'css/footable.metro.css'
```
Comment (jace) [inline funnel/__init__.py]: Is this an external library? It should be in Baseframe then, with the correct upstream version number. `version` here refers to Funnel's version number.

**Comment by jace** [comment]
Apart from moving Footable to Baseframe, this PR looks okay to me.

**Comment by shreyas-satish** [comment]
Can we customize the color for the table header? It's currently `#EC9`, it could perhaps reuse an existing color in `_colors.scss`, such as `$color-outline-thin`. Apart from this and moving FooTable to baseframe, this looks good to me.

File: funnel/__init__.py (hunk 1 above)
Comment (vidya-ram) [inline funnel/__init__.py]: I have moved footable to baseframe (branch funnel-ui). Footable is jquery plugin to make tables responsive.

**Comment by vidya-ram** [comment]
Can we merge this ?

**Pull request merged**